frontend/src/components/IntegrationConfiguration.tsx

```diff
@@ -152,24 +152,32 @@ const MapStates: FC<{ configuration: IntegrationConfiguration }> = ({
             {selected && (
               <SetMapping name={selected.name} select={select(selected.id)} />
             )}
-            {columns &&
-              statusMapping?.map(({ id: mappingId, fromColumn, toStatus }) => {
-                const columnName = columns.find(
-                  (column) => column.id === fromColumn,
-                )?.name;
-                if (!columnName) return undefined;
-                return (
-                  <SetMapping
-                    key={fromColumn}
-                    name={columnName}
-                    status={toStatus}
-                    select={select(fromColumn)}
-                    remove={() =>
-                      removeMapping({ roadmapId, name, id, mappingId })
-                    }
-                  />
-                );
-              })}
+            {columns && statusMapping && statusMapping.length > 0 && (
+              <>
+                <div className={classes(css.statusMappingLabel)}>
+                  {t('Selected mappings')}
```

Comment:
Just a thought: would 'Saved mappings' fit the label better? We don't have much to communicate the mappings are being saved (I recall trying to find a save button when I first tried mapping)

In either case, let's add the label to English translations